src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -86,6 +86,11 @@ def load_arguments(self, _):
         arg_type=get_three_state_flag(), min_api='2018-06-01',
         help='Enables or disables the capability to have 1 or more managed data disks with UltraSSD_LRS storage account')
 
+    scale_in_policy_type = CLIArgumentType(
+        nargs='+', arg_type=get_enum_type(self.get_models('VirtualMachineScaleSetScaleInRules')),
+        help='Specify the scale-in policy (space delimited) that decides which virtual machines are chosen for removal when a Virtual Machine Scale Set is scaled-in.'
```

Comment:
I prefer using "specifies" rather than "decides".

Reply:
I copy it from swagger